packages/nuxt/src/core/plugins/virtual.ts

```diff
@@ -0,0 +1,74 @@
+import { findPath, resolveAlias, useNuxt } from '@nuxt/kit'
+import { dirname, isAbsolute, resolve } from 'pathe'
+import { createUnplugin } from 'unplugin'
+
+const PREFIX = '\0virtual:nuxt:'
+
+interface VirtualFSPluginOptions {
+  mode: 'client' | 'server'
+  alias?: Record<string, string>
+}
+
+export const VirtualFSPlugin = (nuxt = useNuxt(), options: VirtualFSPluginOptions) => createUnplugin(() => {
```

Comment:
```suggestion
export const VirtualFSPlugin = (nuxt, options: VirtualFSPluginOptions) => createUnplugin(() => {
```

Isn't this redundant? Since you don't normalize the options down the line inside the function, you'll always have to pass the `options` argument, which require you to pass the `nuxt` argument first.